[CN-1101] Update Adding Custom Jar Page (#986)
hazelcast/hz-docs: docs/modules/kubernetes/pages/helm-adding-custom-jar.adoc

= Adding custom JAR files to the Platform/Management Center Classpath

> NOTE: We highly recommend you to use Hazelcast Platform Operator's xref:operator:ROOT:user-code-deployment.adoc[User Code Deployment] feature if you need to add custom jar to your Hazelcast members.

Reviewer: ```suggestion
NOTE: We strongly recommend that you use Hazelcast Platform Operator's xref:operator:ROOT:user-code-deployment.adoc[User Code Deployment] feature if you need to add a custom JAR to your Hazelcast members.
```

> If you'd like to add custom JAR files to the Platform/Management Center, you can extend the Hazelcast/Management Center base image and provide your configuration file or/and custom JARs.

Reviewer: ```suggestion
If you want to add custom JAR files to the Platform and/or Management Center, you can extend the the relevant Center base image and provide your configuration file and/or custom JARs.
```

> To do that, you need to create a new `Dockerfile` and build it with `docker build` command.

Reviewer: ```suggestion
To do this, you must create a new `Dockerfile` and build it using the `docker build` command.
```

> In the `Dockerfile` example below, we are creating a new image based on the Hazelcast image and adding a custom JAR

Reviewer: ```suggestion
In the `Dockerfile` example below, we create a new image based on the Hazelcast image and add a custom JAR
```
